airflow/contrib/hooks/gcp_dataflow_hook.py

```diff
@@ -628,12 +628,8 @@ def _set_variables(variables):
 
     # [EWT-1001]: Added appropriate args and fn calls for job naming
     def start_java_dataflow(self, job_name, variables, dataflow, job_class=None,
-        append_job_name=True, cancel_existing_job=False, context=None):
-        self._validate_params(cancel_existing_job, append_job_name)
-        if not append_job_name:
-            # Add execution time to job name if we are not appending a random suffix
-            job_name = self._add_execution_date_to_job_name(job_name, context)
-
+        append_job_name=True, cancel_existing_job=False, track_job=False):
```

Comment:
Are we not appending `execution time` now?

Reply:
We are doing that in the dataflow operators instead of hook so as not to change the hook behaviour.